hypervisor/include/arch/x86/timer.h

```diff
@@ -48,19 +72,57 @@ static inline void initialize_timer(struct hv_timer *timer,
 	}
 }
 
+/**
+ * @brief Check a timer whether expired.
+ *
+ * @param[in] timer Pointer to timer.
+ *
+ * @return true if the timer is expired
+ * @return false if the timer is not expired.
```

Comment:
```suggestion
```

Reply:
I prefer to have all `@return` values under a single `@return` directive, else multiple sections are created in the HTML output and it's not very readable.

Reply:
How about using ''@retval'' instead of ''@return''? Doxygen is able to combine adjacent ''@retval''s into a single paragraph. #1595 has a detailed example.

Reply:
Using `@retval` works for me!